examples/specs/bar_diverging_stacked.vl.json

```diff
@@ -0,0 +1,376 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A diverging stacked bar chart for sentiments towards a set of eight questions, displayed as percentages with neutral responses straddling the 0% mark",
+  "data": {
+    "values":
```

Comment:
Should we be inlining the data in this example? Is there an existing dataset we can use to show diverging stacked bars?

Reply:
The data is specific to this example.

Reply:
Okay! @domoritz perhaps we should factor out and add to https://github.com/vega/vega-datasets. It's kind of verbose to be included in the spec like this IMO

Reply:
Extracting the dataset is a good idea, but we should not do that yet in this PR since this example currently include manually derived fields that would be possible once @AkshatSh add supports for window transform.  (At that point, it is okay to add raw data to the vega-dataset repo.)

Reply:
Agreed with Ham.